src/schema/v2/me/conversation/index.ts

```diff
@@ -134,6 +134,24 @@ const ConversationItem = new GraphQLObjectType<any, ResolverContext>({
     permalink: {
       type: GraphQLString,
     },
+    liveArtworkItem: {
+      type: ConversationItemType,
+      resolve: (artworkItem, _args, { artworkLoader }) => {
+        if (artworkItem.item_type === "Artwork") {
+          return artworkLoader(artworkItem.properties.id).then((artwork) => {
+            const updatedArtwork = {
+              ...artwork,
+              __typename: "Artwork",
+            }
+            return artwork.published ? updatedArtwork : null
+          })
+        } else if (artworkItem.item_type === "PartnerShow") {
+          throw new Error("PartnerShow not supported.")
+        } else {
+          return null
```

Comment:
To be consistent, maybe we should also raise an error if the item type is not supported.

Reply:
Do you mean throw an error in the else block versus return null?

Reply:
I was more so following the [`ConversationItemType`](https://github.com/artsy/metaphysics/blob/2202f0c3491d566bde20351248092af97283b4e2/src/schema/v2/me/conversation/index.ts#L106-L119) pattern. I throw an error for the `PartnerShow` because one might expect for it to be supported due to the typing. If any other item where to come through I think I might expect it to return `null` as is the case with ConversationItemType. This keeps the implementation similar to how `item` works.

@starsirius let me know if you have strong opinion on this though.